src/precompile.jl

```diff
@@ -93,6 +93,28 @@ function pkg_precompile()
                     Pkg.REPLMode.try_prompt_pkg_add(Symbol[:notapackage])
                     Pkg.update(; update_registry=false)
                     Pkg.status()
+                    pkgs_path = abspath(joinpath(dirname(pathof(Pkg)), ".." ,"test", "test_packages"))
+                    # Precompile a diverse set of test packages
+                    # Check all test packages occasionally if anything has been missed
+                    # test_packages = readdir(pkgs_path)
+                    test_packages = (
+                        "ActiveProjectInTestSubgraph",
+                        "BasicSandbox",
+                        "DependsOnExample",
+                        "PackageWithDependency",
+                        "SameNameDifferentUUID",
+                        "SimplePackage",
+                        "BasicCompat",
+                        "PackageWithDependency",
+                        "SameNameDifferentUUID",
+                        "SimplePackage"
```

Comment:
Probably worth doing one with weakdeps
```suggestion
                        "SimplePackage",
                        joinpath("ExtensionExamples", "HasExtensions.jl")
```